index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Linkbrary</title>
+    <meta property="og:title" content="Linkbrary" key="title" />
+    <meta
+      name="description"
+      property="og:description"
+      content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요"
+      key="description"
+    />
+    <meta
+      property="og:image"
+      content="./images/library.PNG"
+      key="image"
+    />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@linkbrary" />
+    <meta name="twitter:title" content="Linkbrary" />
+    <meta name="twitter:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta
+      name="twitter:image"
+      content="./images/library.PNG"
+    />
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+      <nav> 
+        <div class="toolbar">
+          <a href="index.html">
+            <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
+          </a>
+          <a class="login" href="signin.html">
+            <span>로그인</span>
+          </a>
+        </div>
+      </nav> 
+
+
+        <div class="bg">
+            <div class="title">
+                <span class="grad">세상의 모든 정보</span><span class="black">를<br>쉽게 저장하고 관리해 보세요</span>
+            </div>
+
+            <div class="addlink" onclick="location.href='/signup.html'">
+                링크 추가하기
+            </div>
+
+            <div class="img1">
+                <img class="img1_" src="./images/img1.png">
+            </div>
```

Comment:
`img`에 `div` wrapping은 불필요해 보이네요. `.img1_`으로 선언한 클래스만 사용해도 디자인 요구사항을 충분히 구현할 수 있기 때문에 불필요한 요소는 제거하는게 어떨까요?

```suggestion
            <img class="img1_" src="./images/img1.png">
```